### DISCUSSION TITLE
Integration with Cargo.site?

### DISCUSSION THREAD

**Discussion by nweii** [sampotts/plyr#2373] [Q&A]
Hello,

Is anyone here familiar with Cargo.site and how feasibly I could integrate this with my cargo webpage? Cargo allows full access to the website's css stylesheet and limited support for adding custom, site-wide HTML.

I'm not much of a developer/coder, so excuse me if I'm totally out of my depth here.

**Comment by aleriverazz** [comment]
Hi! Did you find any way to integrate it to you GitHub Repo? :(
